com.snipaste.Snipaste.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: com.snipaste.Snipaste
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: snipaste
+separate-locales: false
+
+finish-args:
+  - --device=all
+
+  - --socket=wayland
+  - --socket=fallback-x11
+
+  - --share=ipc
+  - --share=network
+
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-run/pipewire-0
```

Comment:
`xdg-run/pipewire-0` why is this needed?

Reply:
also does it work with `:ro`?

Reply:
removed pipwire. The pictures is used to store the screenshot.